app/assets/stylesheets/_host_next_toc.scss

```diff
@@ -0,0 +1,3 @@
+.form-control {width: 600px; margin: 20px}
```

Comment:
You're styling ALL `form-control` classes that you have. You probably don't want that.

Please add an extra class to the form you want this applied to (and don't assume such a wide screen - treat mobile as a first class citizen!)